examples/repl/cmdline.rs

```diff
@@ -171,8 +171,11 @@ async fn log_msg(context: &Context, prefix: impl AsRef<str>, msg: &Message) {
     let contact = Contact::get_by_id(context, msg.get_from_id())
         .await
         .expect("invalid contact");
-
-    let contact_name = contact.get_name();
+    let contact_name = if let Some(name) = msg.get_override_sender_name() {
+        format!("~{}", name)
+    } else {
+        contact.get_display_name().to_string()
+    };
```

Comment:
You can use `msg.get_sender_name(contact)`.

If/as we want use this "~", we should also change `get_sender_name()`, anyway.

Reply:
`get_sender_name()` is part of the core and we do not want to have such an "UI" element in the core. also, there are reasons, not to add the `~` - eg. on android, currently, we add that in front of the colored name in the bubbles - but not in the notification, not in the chatlist etc.

until that is more settled, i'd keep the flexibility and treat get_override_sender_name() as a "flag" and display things differently if we get sth. back.